### PR TITLE
New checkpoint for the May 15th 2018 protocol upgrade

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

**Comment by sickpig** [comment]
rebased

File: src/chainparams.cpp [hunk 1]
```diff
@@ -197,9 +197,11 @@ class CMainParams : public CChainParams
...
                 478559, uint256S("0x000000000000000000651ef99cb9fcbe0dadde1d424bd9f15ff20136191a5eec"))(
                 // November 13th 2017 new DAA fork
-                504031, uint256S("0x0000000000000000011ebf65b60d0a3de80b8175be709d653b4c1a1beeb6ab9c")),
-            1483472411, // * UNIX timestamp of last checkpoint block
-            184495391, // * total number of transactions between genesis and last checkpoint
+                504031, uint256S("0x0000000000000000011ebf65b60d0a3de80b8175be709d653b4c1a1beeb6ab9c"))(
+                // May 15th 2018 re-enable op_codes and 32 MB max block size
+                530359, uint256S("0x0000000000000000011ada8bd08f46074f44a8f155396f43e38acf9501c49103")),
+            1526410186, // * UNIX timestamp of last checkpoint block
+            249416375, // * total number of transactions between genesis and last checkpoint
```
Comment (ptschip) [inline src/chainparams.cpp:204]: i think the total num transactions is a little high...this is what i get from txoutsetinfo


06:07:29
￼
{
  "height": 531175,
  "bestblock": "00000000000000000172f0cbdead4312dc4e0d42dcec3ea87b8f996d1588ece5",
  "transactions": 17862068,
  "txouts": 39837658,
  "hash_serialized_2": "c112904955a45b764aa5dbc18469b5d7dfe474da1ea2648fa454c4f8ff381487",
  "disk_size": 2241613078,
  "total_amount": 17139535.94435441
}

**Comment by sickpig** [comment]
Isn't the number of total txns since genesis block the one reported in the `UpdateTip` log line? 

```2018-05-15 18:49:56 UpdateTip: new best=0000000000000000011ada8bd08f46074f44a8f155396f43e38acf9501c49103  height=530359 bits=402769355 log2_work=87.290889  tx=249416375  date=2018-05-15 18:49:46 progress=1.000000  cache=0.3MiB(2393txo)```

i.e. `chainActive.Tip()->nChainTx`

if instead we calculate the avg number of transactions based on this assumption we get an avg of `470` txs per block, whereas if we consider the number reported by `gettxoutsetinfo` we get something like `33` txns per block on avg.

It seems pretty low to me but I could be wrong.

**Comment by ptschip** [comment]
i don't know, i didn't see that.... you math sounds right, but i guess
there's a bug somewhere , maybe in gettxoutsetinfo?

**Comment by sickpig** [comment]
@ptschip if there's a bug in the code I think it is present in Core, just checked and they are at height`523699` the txs count contained `UpdateTip` is `317638309`, whereas the results of `gettxoutsetinfo` is:

```
$> bitcoin-cli gettxoutsetinfo
{
  "height": 523699,
  "bestblock": "00000000000000000027c20bb9c55ea0aa976b2f4024cf05d25cf2e908f7b89b",
  "transactions": 24367283,
  "txouts": 51600481,
  "bogosize": 3881977444,
  "hash_serialized_2": "8475948c7ad51f6728f94c2bb4c0b41fe6b58426713fd8a44d04bedf925f302e",
  "disk_size": 2937214127,
  "total_amount": 17046073.58574235
}
```

**Comment by sickpig** [comment]
@ptschip no bug.

checked the code and the txns count reported by gettxoutsetinfo is the number of transactions that refer to at least one non-spent output in the UTXO set.

**Review by ptschip — APPROVED** [review]
great, thanks for checking on that txoutsetinfo.

**Pull request merged**